### PR TITLE
Add new like

### PR DESCRIPTION
**Pull request by Timbar09**
I implemented adding a new like to the items and cleaned up the code a bit in this PR.

**Changes**

- Implemented functions fetching, receiving, and displaying likes
- Moved API-related code to the api.js file
- Added an animation on the like icon

### CODE REVIEW

**Review by oyhpnayiaw — APPROVED** [review]
Good job @Timbar09 ! Very cute ❤️ animation and code are more cleaner than before. But I have [optional] Suggestion. See comments. If you want to keep the way it is you can do that and merge it.

File: src/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { createPopup } from './modules/popup.js';
 
 const showsContainer = document.querySelector('.maze__grid');
 
-showsContainer.addEventListener('click', (e) => {
+showsContainer.addEventListener('click', async (e) => {
   const commentBtn = e.target.closest('.comment__btn');
```
[the remark this answers is not in the document]
Comment (Timbar09) [reply src/index.js:9]: You read my mind, @oyhpnayiaw. I'm on it.

[the remark this answers is not in the document]
Comment (Timbar09) [reply src/index.js]: It's done, @oyhpnayiaw.

[the remark this answers is not in the document]
Comment (oyhpnayiaw) [reply src/index.js]: Good bro, you can merge now

**Pull request merged**